app/src/scenes/public-support-center/form.js

```diff
@@ -128,23 +158,28 @@ export default function FormComponent({ setOpen, setSuccessMessage }) {
   );
 }
 
-const Item = ({ title, name, value, handleChange, errors, touched, validate, type, options, ...props }) => {
+const Item = ({ title, name, value, handleChange, errors, touched, validate, type, options, placeholder, ...props }) => {
   return (
     <Col style={{ marginTop: 20 }}>
       <Label>{title}</Label>
       {type === "select" ? (
         <Field as={type} className="form-control" name={name} value={value} onChange={handleChange} validate={validate} {...props}>
           <option value="" disabled>
```

Comment:
Peut etre remettre ca ici aussi ? (tu l'as mis dans le `SelectTag` je trouvais ca cool.
Sinon, par défaut, c'est le premier departement qui est "selectionné", alors qu'il ne l'est pas vraiment
```suggestion
          <option disabled value="" selected={!value} label={placeholder}>
```